CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+## 0.1.2
+  * Fall back to default values for un-configured optional params [#25](https://github.com/singer-io/tap-snapchat-ads/pull/25)
+  * Fix `End time should be after start time` error [#26](https://github.com/singer-io/tap-snapchat-ads/pull/26)
+  * Fix URL creation using `urlencode` [#26](https://github.com/singer-io/tap-snapchat-ads/pull/26)
```

Comment:
* Fixes following issues [#26](https://github.com/singer-io/tap-snapchat-ads/pull/26)
    * `End time should be after start time` error
    * URL creation using `urlencode

    You can put something like this, if you are fixing multiple issues in single PR, insread of repeating PR link at multiple times.